model/client.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+
+const uuidv4 = require('uuid-v4');
+const Client = module.exports = (socket) => {
```

Comment:
This is throwing an error that Client is not a constructor. I was able to fix it by changing it back to the pattern used in class by Brian. Switch it back to the word function to get this to work.